hircluster.h

```diff
@@ -267,8 +262,10 @@ int redisClusterGetReply(redisClusterContext *cc, void **reply);
 /* Reset context after a performed pipelining */
 void redisClusterReset(redisClusterContext *cc);
 
+/* Update the slot map by querying any node. */
+int redisClusterUpdateSlotMap(redisClusterContext *cc);
```

Comment:
SlotMap or Slotmap?

There's a field called `lastSlotmapUpdateAttempt` in redisClusterAsyncContext. Other than that, all slotmap functions are internal, so maybe stick with Slotmap in the public API?

This also controls whether it is one word "slotmap" or two "slot map" in documentation. So far, we use "slotmap" and the event name `HIRCLUSTER_EVENT_SLOTMAP_UPDATED` also indicates slotmap is one word.

Reply:
Slotmap in one single word is good, my first choice until the internal `updateSlotMapAsync()` triggered a change.